testing/tests/inheritance.rs

```diff
@@ -311,14 +311,9 @@ fn test_flat_deep() {
 
 #[derive(Template)]
 #[template(path = "let-base.html")]
+#[allow(dead_code)]
```

Comment:
The commit message doesn't quite make sense here, maybe clean that up?

Reply:
True, it's not a clippy lint but new rustc warnings. Gonna update the commit message.